CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ find_package(Chaste COMPONENTS heart)
 
 # Here we add extra arguments to force PyCML to use this extra argument (make Get and Set methods for
 # all metadata annotated variables). 
-set(Chaste_PYCML_EXTRA_ARGS "--expose-annotated-variables")
```

Comment:
So the `--expose-annotated-variables` flag has disappeared because it's now the default has it? Is there an option to switch this off (could have some very small speed improvement I guess).

Reply:
It's always on.
The flag still exists (hidden) for backwards compatibility, but has no effect.

Reply:
OK, I think I can live with that :-)